frontend/packages/db/supabase/migrations/20250410121418_add_project_table_rls.sql

```diff
@@ -0,0 +1,76 @@
+ALTER TABLE "public"."Project" ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY "authenticated_users_can_select_org_projects" ON "public"."Project"
+FOR SELECT
+TO authenticated
+USING (
+  "organizationId" IN (
+    SELECT "organizationId" 
+    FROM "public"."OrganizationMember" 
+    WHERE "userId" = auth.uid()
+  )
+);
+
+CREATE POLICY "authenticated_users_can_insert_projects" ON "public"."Project"
+FOR INSERT
+TO authenticated
+WITH CHECK (true);
```

Comment:
I'm not 100% sure, but I think there might be a small concern with the current INSERT policy.

Since there's no check for whether the user belongs to the organization, it might allow someone to create a project under an org they’re not part of.

Maybe adding a WITH CHECK like this could help? (postgres can this?)

```suggestion
WITH CHECK (
  "organizationId" IN (
    SELECT "organizationId" 
    FROM "public"."OrganizationMember" 
    WHERE "userId" = auth.uid()
  )
);
```

Reply:
[🐛 fix: update RLS policy for project insertion to restrict by organiz…](https://github.com/liam-hq/liam/pull/1304/commits/6148eb32c7d61087d53e144ab512b239048e6d91)
The approach you suggested worked perfectly, thank you 🙏
I was a bit concerned that the RLS policy might not work for INSERTs since the record hadn’t been created yet, but that turned out to be unnecessary worry.
I've confirmed it works as expected locally.